.eslintrc.json

```diff
@@ -12,14 +12,16 @@
     "ecmaVersion": 2018,
     "sourceType": "module"
   },
-  "extends": ["eslint:recommended", "plugin:react/recommended", "airbnb"],
+  "extends": ["eslint:recommended", "plugin:react/recommended", "airbnb", "plugin:prettier/recommended"],
   "settings": {
     "react": {
       "version": "detect"
     }
   },
-  "plugins": ["react", "jsx-a11y", "import"],
```

Comment:
we have some `import/...` rules, don't think we should remove "import" plugin

Reply:
I think I removed that thinking it was part of a certain plugin that I'd thought would fix the graphql issue. I'll add it back in then